packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -207,6 +218,10 @@ class HoldScrollActivity extends ScrollActivity implements ScrollHoldController
 ///  * [DragScrollActivity], which is the activity the scroll view performs
 ///    while a drag is underway.
 class ScrollDragController implements Drag {
+  /// Maximum amount of time the drag can remain stationary before losing
+  /// the momentum carried from a previous scroll activity.
+  static const Duration momentumRetainStationaryThreshold = const Duration(milliseconds: 20);
+
```

Comment:
Move this below the constructor.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-class-members-by-typical-lifecycle

Reply:
moved below instance variables like other files. 

Tweaked the doc for a bit more clarification.